ReactApplication/src/Components/ColorfulGrid.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from "react";
+import { SketchPicker } from "react-color";
+import "./style.css";
+import { getRandomColor } from "../Helpers/Helpers";
+
+const INITIAL_COLORS = getRandomColor(5);
+const ColorfulGrid = () => {
+  const [color, setColor] = useState("#ffffff");
+  const [colors, setColors] = useState(INITIAL_COLORS);
+  const [rows, setRows] = useState(3);
+  const [columns, setColumns] = useState(3);
+
+  // handlers
+  function onDelete(index) {
+    setColors((prev) => {
+      let remainColors = [];
+      remainColors = prev.filter((c, i) => i !== index);
+      return remainColors;
+    });
+  }
+
+  function colorChanhgeHandler(index, newColor) {
+    setColors((prev) => {
+      let upadtedColors = [];
+      upadtedColors = upadtedColors.concat(prev);
+      upadtedColors[index] = newColor;
+      return upadtedColors;
+    });
+  }
+  //
+  const renderGrid = () => {
+    const grid = [];
+    const numberOfColors = colors.length;
+    for (let i = 0; i < rows; i++) {
+      for (let j = 0; j < columns; j++) {
+        const randomNumber = Math.floor(Math.random() * numberOfColors);
+        const randomColor = colors[randomNumber];
+        grid.push(
+          <div
+            key={`${i}-${j}`}
+            className="grid-item"
+            style={{ backgroundColor: randomColor }}
+          >
+            
+          </div>

```

Comment:
```suggestion
          <div
            key={`${i}-${j}`}
            className="grid-item"
            style={{ backgroundColor: randomColor }}
          />
```
can we implement the same functionality without using inline CSS.